Lib/test/libregrtest/save_env.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import threading
 
+from importlib import import_module
```

Comment:
Nit, re: where the whitespace goes

```suggestion
from importlib import import_module

```

I see the existing sections as stdlib, test, and local rather than full imports, from imports, and local.

Reply:
I'm going to change this to import `termios` at the top level, but wrapped in a try/except.  The reason being, I think it's better to attempt the import once and then the saved environments can use it if available, rather than try to import it every time we try to save the environment.  I'm going to get rid of the `demand` argument too.  `termios` is Unix only.